### PR TITLE
feat: add GetFormResponse domain

### LINKED ISSUE #186: フォームの取得

### PR DESCRIPTION
**Pull request by dino3616**
- close #186 

## 参照ドキュメント

https://laminne.stoplight.io/docs/enshita/9c08d51410814-

### CODE REVIEW

**Comment by dino3616** [comment]
ready for review

File: server/model/domain/get_form.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package domain
+
+import "time"
+
+type GetFormResponse struct {
+	ID        string    `json:"id"`
+	Title     string    `json:"title"`
+	Deadline  time.Time `json:"deadline"`
```
Comment (Sigumaa) [inline server/model/domain/get_form.go:8]:
```suggestion
	DeadLine  time.Time `json:"deadLine"`
```

File: server/model/domain/get_form.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+		EMail            string `json:"email"`
+		Title            string `json:"title"`
+		Paragraph        string `json:"paragraph"`
+		SpeakerQuotaType []struct {
+			ID           string    `json:"id"`
+			Name         string    `json:"name"`
+			Time         time.Time `json:"time"`
+			CurrentCount int       `json:"current_count"`
+			Total        int       `json:"total"`
+		} `json:"speaker_quota_type"`
```
Comment (Sigumaa) [inline server/model/domain/get_form.go:21]:
```suggestion
		} `json:"speakerQuotaType"`
```

File: server/model/domain/get_form.go [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	Questions struct {
+		Name             string `json:"name"`
+		EMail            string `json:"email"`
+		Title            string `json:"title"`
+		Paragraph        string `json:"paragraph"`
+		SpeakerQuotaType []struct {
+			ID           string    `json:"id"`
+			Name         string    `json:"name"`
+			Time         time.Time `json:"time"`
+			CurrentCount int       `json:"current_count"`
```
Comment (Sigumaa) [inline server/model/domain/get_form.go:19]:
```suggestion
			CurrentCount int       `json:"currentCount"`
```

**Review by Sigumaa — CHANGES_REQUESTED** [review]

**Review by Sigumaa — APPROVED** [review]

File: server/model/domain/get_form.go [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import "time"
+
+type GetFormResponse struct {
+	ID        string    `json:"id"`
+	Title     string    `json:"title"`
+	DeadLine  time.Time `json:"deadLine"`
+	Enable    bool      `json:"enable"`
+	Questions struct {
+		Name             string `json:"name"`
+		EMail            string `json:"email"`
```
Comment (Sigumaa) [inline server/model/domain/get_form.go:12]:
```suggestion
		Email            string `json:"email"`
```

**Review by Sigumaa — CHANGES_REQUESTED** [review]

**Review by Sigumaa — APPROVED** [review]

**Review by laminne — APPROVED** [review]

**Comment by dino3616** [comment]
working state...

**Review by Sigumaa — APPROVED** [review]
***Approve***

**Review by laminne — APPROVED** [review]

**Pull request merged**